modules/contact/src/constraints/ComputeDynamicWeightedGapLMMechanicalContact.C

```diff
@@ -206,3 +241,42 @@ ComputeDynamicWeightedGapLMMechanicalContact::incorrectEdgeDroppingPost(
     ComputeWeightedGapLMMechanicalContact::enforceConstraintOnDof(pr.first);
   }
 }
+
+void
+ComputeDynamicWeightedGapLMMechanicalContact::communicateWear()
```

Comment:
at some point we might think about consolidating all these communication methods into a single templated method that can be called for all the use cases

Reply:
Yeah, that sounds good. It's been used already for 3 different quantities. Will keep it in mind for next time.